grafana/dashboards/cmode/workload.json

```diff
@@ -201,7 +201,7 @@
       "targets": [
         {
           "exemplar": false,
-          "expr": "(avg(qos_detail_volume_resource_latency{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",workload=~\"$Workload\",volume=~\"$Volume\"}) by (resource))",
+          "expr": "(avg(qos_detail_resource_latency{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",workload=~\"$Workload\"}) by (resource))",
```

Comment:
we don't need value change based on volume filter?
something like: `volume!=\"\", volume=~\"$Volume\"`